test/Microsoft.AspNetCore.Mvc.Core.Test/ModelBinding/Binders/ParsingSimpleTypeModelBinderProviderTest.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Mvc.ModelBinding.Binders
+{
+    public class ParsingSimpleTypeModelBinderProviderTest
```

Comment:
The new test classes started as copies of their non-Parsing sister (`SimpleTypeModelBinderProviderTest` in this case). I likely left a few duplicates behind in `SimpleTypeModelBinderIntegrationTest` but am not too worried about it.